src/_userflair.scss

```diff
@@ -17,16 +17,27 @@
 }
 
 // List service icons
-.flair:before {
+.flair::before {
 	content: "";
-	width: 16px;
-	height: 16px;
+	width: 0px;
+	height: 0px;
 	background: url("images/general.png") 0 0/128px;
 	border-radius: inherit;
 	margin-left: -16px;
 	transition: inherit;
 }
 
+.flair-MAL::before, 
+.flair-HB::before,
+.flair-ANI::before,
+.flair-AP::before,
+.flair-ann::before,
+.flair-anidb::before,
+.flair-custom::before {
+	width: 16px;
+	height: 16px;
+}
```

Comment:
Just out of curiosity, will these be replaced with subreddit emoji at some point as well?